woocommerce/woocommerce
# change the namespace of the WC Admin Package class

### All Submissions:


### Changes proposed in this Pull Request:

This PR changes the namespace of the WC Admin `Package` class to prevent the WC Admin autoloader from providing the class when WC Admin is active as a standalone plugin.

Merging should wait until https://github.com/woocommerce/woocommerce-admin/pull/3687 is merged and WC Admin 0.26.0 is released. `composer.json` has been updated to `0.26.0`.

Closes #25700 .

### How to test the changes in this Pull Request:

1. Run `composer install`
2. Activate WC Admin 0.26
3. Edit the `VERSION` constant in `packages/woocommerce-admin/src/Packages.php` to be `0.27.0`
4. Navigate to any WC Admin enabled page
5. See note recommending deactivating WC Admin
6. Click to any non WC page
7. Edit the `VERSION` to be `0.25.0` 
8. Navigate to any WC Admin enabled page
9. No deactivation note should be shown

### Other information:
